sourced/ml/__main__.py

```diff
@@ -192,6 +192,22 @@ def add_parser(name, help_message):
         "input", help="Path to the model file, URL or UUID.")
     dump_parser.add_argument("--gcs", default=None, dest="gcs_bucket",
                              help="GCS bucket to use.")
+    # ------------------------------------------------------------------------
+    merge_df = add_parser("merge_df", "Merge DocumentFrequency models to a singe one.")
+    merge_df.set_defaults(handler=merge_df_entry)
+    add_filter_arg(merge_df)
+    add_min_docfreq(merge_df)
+    add_vocabulary_size_arg(merge_df)
+    merge_df.add_argument(
+        "-o", "--output", required=True,
+        help="Path to the merged document frequencies model.")
+    merge_df.add_argument(
```

Comment:
How about taking a list of files as an input? Recursive scan can easily be performed with `find` and the whole thing can be run as

```
find '*.asdf' | srcml mdf -i - -o result.asdf
```

Otherwise it is impossible to merge only a few specific files, only the directory.

Reply:
Actually, you can do it with `--filter` flag.
Maybe it is better to keep both options? 
you can specify `-i -` to read a list of files from stdin or provide a directory `-i dirpath`

Reply:
This is called the "UNIX way" :) You can remove all that code and do not support it with tests.
Anyway, the stdin feature should be here.